static/js/hotkey.js

```diff
@@ -448,6 +453,11 @@ exports.process_hotkey = function (e, hotkey) {
         return false;
     }
 
+    if (popovers.reactions_popped()) {
+        reactions.reaction_navigate(e, event_name);
+        return false;
+    }
```

Comment:
This should `return true` if and only if `reactions.reaction_navigate` returns true (well, depends a bit on whether we want hotkeys like `?` to work with the popover open, but we probably do?).  And then `reactions.reaction_navigate` won't need its own `preventDefault()` (and you won't need to pass `e` to the function).  In general, every hotkey that definitively knows what was pressed should return true here.